packages/flame/lib/src/components/component.dart

```diff
@@ -319,6 +319,22 @@ class Component {
     }
   }
 
+  /// An iterator that returns all added components.
```

Comment:
I think this can be improved. Here is a suggestion, not sure about it, but something along the lines could be nice.

```suggestion
  /// Deeply searches all [children].
```

Reply:
Totally agree with you)

Reply:
done